taxi/views.py

```diff
@@ -87,3 +89,37 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+    form_class = DriverLicenseUpdateForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+def remove_driver_from_car(request, pk):
```

Comment:
You can put logic from `remove_driver_from_car` and `assign_driver_to_car` into post method of `CarDetailView`. The reason is that we don't need to create two extra views and two extra endpoints to handle adding/deleting driver.
So the implementation will look something like this:

1. Add an empty form instead of links on car detail template that will send value 'Add' or 'Delete'
2. This form will send a request to CarDetailView (so no need to make extra endpoints) in a post method
3. You override post method in this view and catch those values that you received in if condition and add/remove driver to car through related fields.
4. You will need to figure out how to get car and user in the view's post method to set related fields.